test/helpers/kubectl.go

```diff
@@ -1519,9 +1519,12 @@ func (kub *Kubectl) waitToDeleteCiliumOperator() error {
 		default:
```

Comment:
At the line 1518 (GH does not allow me to comment it), you probably want to replace `"Cilium Operator"` with `name`.